cmd/dastard/dastard.go

```diff
@@ -91,6 +91,6 @@ func main() {
 		panic(err)
 	}
 
-	go dastard.RunClientUpdater(dastard.Ports.Status)
+	go dastard.RunClientUpdater(dastard.Ports.Status, nil)
```

Comment:
This argument was to allow testing code to abort the client updater (which gets ride of a zmq message about "dangling 'PUB' socket'). It's not needed in regular dastard, though I suppose it could be added.

Reply:
Okay, I added it to the main executable just to be complete.